### DISCUSSION TITLE
Does Directus supports microservice architecture???

### DISCUSSION THREAD

**Discussion by sanjeevbrok99** [directus/directus#15942] [Archive]
Hi @freekrai Can we use directus with microservice applications??

**Comment by rijkvanzanten** [comment]
Yes. Directus could talk to your microservices through (web)hooks/flows, or you can use Directus as one of the microservices in your organization

**Reply by vitalijalbu** [reply]
But can we create multiple microservices using directus? How it can be? Every microservice a different installation? Or just one mega installation and many extensions?
